analyzer/codechecker_analyzer/buildlog/log_parser.py

```diff
@@ -1289,6 +1288,7 @@ def parse_unique_log(compilation_database,
             uniqueing_re = re.compile(compile_uniqueing)
 
         skipped_cmp_cmd_count = 0
+        __contains_no_intrinsic_headers.cache_clear()
```

Comment:
What is the purpose of clearing the cache in the beginning?

Reply:
In fact it's only used for tests as they share a static cache storage.
In real scenarios this function is called only once so clearing the cache is nop.